strawberryfields/apps/graph/resize.py

```diff
@@ -308,6 +308,18 @@ def clique_swap(clique: list, graph: nx.Graph, node_select: str = "uniform") ->
     supported. Degree-based node selection involves picking the node with the greatest degree,
     with ties settled by uniform random choice.
 
+    Example usage:
+
+    .. code-block::
+
+        >>> from strawberryfields.apps.graph import resize
+        >>> import networkx as nx
+        >>> graph = nx.wheel_graph(5)
+        >>> graph.remove_edge(0, 4)
+        >>> subgraph = [0, 1, 2]  # these nodes form a clique
+        >>> resize.clique_swap(subgraph, graph)
+        [0, 1, 4]
```

Comment:
```suggestion
        [0, 2, 3]
```
When I ran this (multiple times), this is what I got 🤔 
This makes sense, since `c_1 = [(1, 3)]` so we swap out node 1 for node 3.

Reply:
Which is a great motivation to use doctest 😁

Reply:
Good catch! I guess I labelled the nodes differently than networkx. Indeed, good motivation for doctest